build.gradle

```diff
@@ -273,6 +297,9 @@ minecraft {
                     sources(sourceSets.tools, sourceSets.datagentools)
                 }
             }
+
+            //For Mixin in other mods / Patchouli
+            property 'mixin.env.disableRefMap', 'true'
```

Comment:
This can be removed because we already have it above the mods block for datagen

Reply:
Well, no, this is not datagen... Patchouli is added as a mod for testing purposes

Reply:
no I mean the datagen block
```
data {

}
```

Reply:
Doh, sorry, blame mobile github